transfers/amclient.py

```diff
@@ -382,46 +394,77 @@ def hide_transfer(self, transfer_uuid):
             method='DELETE'
         )
 
+    def hide_unit(self, unit_uuid, unit_type):
```

Comment:
Is hide_transfer still needed? If so, can it be converted into a wrapper for hide_unit?